_haikus/open_space.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Open Space
+author: sap143
+---
+
+Simple pull request<br>
+Makes me learn<br>
```

Comment:
The second line of a haiku needs to be 7 syllables long, this is currently 3 syllables.